docs/generate.jl

```diff
@@ -0,0 +1,25 @@
+# generate examples
+import Literate
+
+EXAMPLEDIR = joinpath(@__DIR__, "..", "models")
+GENERATEDDIR = joinpath(@__DIR__, "src", "models")
+mkpath(GENERATEDDIR)
+for example in readdir(EXAMPLEDIR)
+    if endswith(example, ".jl")
+        input = abspath(joinpath(EXAMPLEDIR, example))
+        script = Literate.script(input, GENERATEDDIR)
+        code = strip(read(script, String))
+        mdpost(str) = replace(str, "@__CODE__" => code)
+        Literate.markdown(input, GENERATEDDIR, postprocess = mdpost)
+        Literate.notebook(input, GENERATEDDIR, execute = true)
+    elseif any(endswith.(example, [".png", ".jpg", ".gif"]))
+        cp(joinpath(EXAMPLEDIR, example), joinpath(GENERATEDDIR, example); force=true)
+    else
+        @warn "ignoring $example"
+    end
+end
+
+# remove any .vtu files in the generated dir (should not be deployed)
```

Comment:
i think this lines 22-25 are not relevant here and we can remove them (or comment)